lib/tarantool_server.py

```diff
@@ -449,7 +449,7 @@ def seek_wait(self, msg, proc=None, name=None):
         while True:
             if os.path.exists(self.path):
                 break
-            time.sleep(0.001)
+            gevent.sleep(0.001)
```

Comment:
There are other `time.sleep()` usages across the code. Can them lead to problems of the same kind? Say, `time.sleep(0.1)` in `wait_until_started()` — I guess it is easy to verify by removing a `console.listen()` call from the instance file.

I don't push you to reproduce and/or investigate all possible problems at once, but, please, look around and if you are not sure that there are no problems anymore, file a follow up issue. If you'll do, leave a link to #276 and back to this discussion.

Reply:
I was investigating it. Tbh, I tried your method but didn't understand clearly the situation. Can we return to this later?

Reply:
Thank you for this reproduce way.

Reply:
How about adding this test case? It is about the {time -> gevent}.sleep() around ECONNREFUSE check.

Reply:
Let's add the TCP binary port case to #274 and resolve it again.

Reply:
Added.